annubes/tasks/task.py

```diff
@@ -16,24 +16,26 @@ class Task:
 
     Args:
         name: Name of the task.
-        session: Dictionary representing the ratio (values) of the different trials (keys) within the task. Trials with
-            a single modality must be represented by single characters, while trials with multiple modalities are
-            represented by the character combination of those trials. Note that values are read relative to each other,
-            such that e.g. `{"v": 0.25, "a": 0.75}` == `{"v": 1, "a": 3}` is True. Defaults to {"v": 0.5, "a": 0.5}.
-        stim_intensities: List of possible intensities of each stimulus.
+        session: Structure of the type of trials which can appear during a session.
```

Comment:
I don't like using the word "type" here, as it has a very specific meaning in python, which is not the what we mean here. Also, it is not really a structure.

```suggestion
        session: Configuration of the trials that can appear during a session.
```